recipe/meta.yaml

```diff
@@ -28,7 +28,8 @@ source:
     - 0018-Use-LAPACK_LDFLAGS-in-Rlapack_la_LIBADD.patch
 
 build:
-  number: 6
+  skip: true  # [win]
```

Comment:
Note [Windows is already disabled in `main`]( https://github.com/conda-forge/r-base-feedstock/blob/6a79d65c6813758360bd884020ac1106471cf470/recipe/meta.yaml#L31 ) due to issue ( https://github.com/conda-forge/r-base-feedstock/issues/214 ). Given this issue would be seen with any Windows build of `r-base`, went ahead and just disabled here until that issue can be resolved.